lib/finders/worldwide_organisations.json

```diff
@@ -0,0 +1,30 @@
+{
+  "base_path": "/government/world/organisations",
+  "title": "Worldwide organisations",
+  "description": "",
+  "locale": "en",
+  "document_type": "finder",
+  "schema_name": "finder",
+  "publishing_app": "whitehall",
+  "rendering_app": "finder-frontend",
+  "details": {
+    "document_noun": "organisation",
+    "default_order": "title",
+    "facets": [],
+    "filter": {
+      "format": "worldwide_organisation"
+    },
+    "format_name": "Worldwide organisations",
```

Comment:
This field is not necessary:

https://github.com/alphagov/finder-frontend/blob/master/docs/finder-content-item.md#format_name

> Not specifically used by the Finder, but used by Specialist Frontend to link back to the Finder. Usually a singularised version of the title of the Finder - "Competition and Markets Authority case" for /cma-cases for example. However there are edge cases where it's not the same such as "Medical safety alert" for Alerts and recalls for drugs and medical devices.